src/aggregator/aggregator/aggregator.go

```diff
@@ -235,6 +242,44 @@ func (agg *aggregator) AddForwarded(
 	return nil
 }
 
+func (agg *aggregator) AddPassThrough(
+	metric aggregated.Metric,
+	metadata metadata.TimedMetadata,
+) error {
+	callStart := agg.nowFn()
+	agg.metrics.passThrough.Inc(1)
+
+	if agg.state != aggregatorOpen {
+		agg.metrics.addPassThrough.ReportError(errAggregatorAlreadyOpenOrClosed)
+		return errAggregatorNotOpenOrClosed
+	}
+	if agg.passThroughWriter == nil {
+		agg.metrics.addPassThrough.ReportError(errPassThroughWriterNotDefined)
+		return errPassThroughWriterNotDefined
+	}
+
+	mp := aggregated.ChunkedMetricWithStoragePolicy{
+		ChunkedMetric: aggregated.ChunkedMetric{
+			ChunkedID: id.ChunkedID{
+				Prefix: nil,
```

Comment:
why are prefix/suffix nil?

if intended, would be good to put a comment explaining thought process

Reply:
Same feedback 😂 also, if you think this would be better expressed through a constructor, maybe add one instead?

Reply:
wrote a comment about it to the best of my understanding. basically we don't want to modify the names for incoming carbon metrics.